apple-dmg/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "apple-dmg"
+version = "0.1.0"
+edition = "2021"
+description = "Library for reading and writing dmg images."
+repository = "https://github.com/indygreg/apple-platform-rs"
+license = "Apache-2.0 OR MIT"
```

Comment:
Could you please copy the pattern used in the `apple-sdk` create to apply the license:

* Add `LICENSE-APACHE` and `LICENSE-MIT` file to the project directory.
* Add a license boilerplate to the top of every `.rs` file:

```
// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
// option. This file may not be copied, modified, or distributed
// except according to those terms.
```